webview-ui/test-generation/src/App.tsx

```diff
@@ -115,26 +121,59 @@ function App() {
     status,
     steps,
     tunnel,
+    loadedAppInfo,
   } = state;
 
+  const loadedAppNames: string[] = loadedAppInfo.map((map) => map.name);
+  const appNameToPlatform = new Map<string, string>();
+  loadedAppInfo.forEach((item) => appNameToPlatform.set(item.name, item.kind));
+  let seenHistoryAppName = appName == ''; // if appName is empty, we don't need to add appName later
+  const optionElements: JSX.Element[] = [];
```

Comment:
Would be great if the app names were sorted. I had a bit of trouble navigating through our relatively large list!

Reply:
Good to know! I'll add that